### ISSUE TITLE
Error while building current version

### ISSUE THREAD

**Issue by janschulte** [52North/sos-importer#30]
Failed to execute goal on project 52n-sos-importer-core: Could not resolve dependencies for project org.n52.sensorweb:52n-sos-importer-core:jar:0.4.0-SNAPSHOT: The following artifacts could not be resolved: org.geotools:gt-swing:jar:9.5, org.geotools:gt-wms:jar:9.5, org.geotools:gt-epsg-hsql:jar:9.5

**Comment by EHJ-52n** [comment]
The build problem is fixed on our build server. The geotools part is still open.

**Comment by johnjohndoe** [comment]
This actually just happens on the *master* branch. If you build *develop* before, all dependencies will be in *~/.m2/repository/*. Which means you can easily reproduce the issue by simply emptying your local Maven repository and then build *master*.

For completeness, here is the console output of `maven clean install` executed on *master*:

```
[INFO] ------------------------------------------------------------------------
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Failed to resolve artifact.

Missing:
----------
1) org.geotools:gt-swing:jar:9.5

  Try downloading the file manually from the project website.

  Then, install it using the command: 
      mvn install:install-file -DgroupId=org.geotools -DartifactId=gt-swing -Dversion=9.5 -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there: 
      mvn deploy:deploy-file -DgroupId=org.geotools -DartifactId=gt-swing -Dversion=9.5 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency: 
  	1) org.n52.sensorweb:52n-sos-importer-core:jar:0.4.0-SNAPSHOT
  	2) org.geotools:gt-swing:jar:9.5

2) org.geotools:gt-wms:jar:9.5

  Try downloading the file manually from the project website.

  Then, install it using the command: 
      mvn install:install-file -DgroupId=org.geotools -DartifactId=gt-wms -Dversion=9.5 -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there: 
      mvn deploy:deploy-file -DgroupId=org.geotools -DartifactId=gt-wms -Dversion=9.5 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency: 
  	1) org.n52.sensorweb:52n-sos-importer-core:jar:0.4.0-SNAPSHOT
  	2) org.geotools:gt-wms:jar:9.5

3) org.geotools:gt-epsg-hsql:jar:9.5

  Try downloading the file manually from the project website.

  Then, install it using the command: 
      mvn install:install-file -DgroupId=org.geotools -DartifactId=gt-epsg-hsql -Dversion=9.5 -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there: 
      mvn deploy:deploy-file -DgroupId=org.geotools -DartifactId=gt-epsg-hsql -Dversion=9.5 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency: 
  	1) org.n52.sensorweb:52n-sos-importer-core:jar:0.4.0-SNAPSHOT
  	2) org.geotools:gt-epsg-hsql:jar:9.5

----------
3 required artifacts are missing.

for artifact: 
  org.n52.sensorweb:52n-sos-importer-core:jar:0.4.0-SNAPSHOT

from the specified remote repositories:
  central (http://repo1.maven.org/maven2),
  geotools-new (http://repo.opengeo.org/),
  n52-snapshots (http://52north.org/maven/repo/snapshots/),
  n52-releases (http://52north.org/maven/repo/releases/)
```

**Comment by EHJ-52n** [comment]
This problem is fixed with be927ba. Please verify with:
```shell
git clone https://github.com/52North/sos-importer.git
cd sos-importer
mvn clean install -Dmaven.repo.local=/tmp/repo-$(date +”%Y%m%d-%H%M%S”)
```
